ansible/prater.yml

```diff
@@ -91,3 +91,27 @@
       loop_control:
         loop_var: node
         index_var: port_offset
+
+- name: Deploy prater macOS beacon nodes
+  become: true
+  remote_user: administrator
+  hosts: nimbus-prater-macos
+  tasks:
+    - include_role: name=infra-role-beacon-node-macos
+      tags: [ beacon-node, infra-role-beacon-node-macos ]
+      vars:
+        beacon_node_repo_branch: '{{ node.branch }}'
+        beacon_node_build_frequency: '{{ node.build_frequency }}'
+        beacon_node_discovery_port: '{{ 9000 + port_offset }}'
+        beacon_node_listening_port: '{{ 9000 + port_offset }}'
+        beacon_node_metrics_port:   '{{ 9200 + port_offset }}'
+        beacon_node_rpc_port:       '{{ 9900 + port_offset }}'
+        beacon_node_dist_validators_start: '{{ node.start }}'
+        beacon_node_dist_validators_end: '{{ node.end }}'
```

Comment:
You can explicitly disable this for now with `beacon_node_dist_validators_enabled: false`.